test/local_sensitivity/concrete_solve_derivatives.jl

```diff
@@ -242,10 +242,10 @@ du02,dp2 = Zygote.gradient(
 
 
 Random.seed!(seed)
-du03,dp3 = Tracker.gradient((u0,p)->sum(concrete_solve(proboop,EulerHeun(),u0,p,dt=1e-2,adaptive=false,save_noise=true,saveat=0.01)),u0,p)
+du03,dp3 = Tracker.gradient((u0,p)->sum(concrete_solve(proboop,EulerHeun(),u0,p,dt=1e-2,adaptive=false,save_noise=true,abstol=1e-14,reltol=1e-14,saveat=0.01)),u0,p)
 
-#Random.seed!(seed)
-du01,dp1 = ReverseDiff.gradient((u0,p)->sum(concrete_solve(proboop,EulerHeun(),u0,p,dt=1e-2,adaptive=false,save_noise=true,saveat=0.01)),(u0,p))
+Random.seed!(seed)
+du04,dp4 = ReverseDiff.gradient((u0,p)->sum(concrete_solve(proboop,EulerHeun(),u0,p,dt=1e-2,adaptive=false,save_noise=true,abstol=1e-14,reltol=1e-14,saveat=0.01)),(u0,p))

```

Comment:
fixed dt methods ignore the tolerances.

Reply:
yep, I thought the error from CI points in this direction: https://travis-ci.org/github/SciML/DiffEqSensitivity.jl/jobs/690732450 "got unsupported keyword arguments "checkpoints", "abstol", "reltol""

A bit strange is that in my local environment with the same tracker version the tracker one at least runs without any error.

Reply:
oh updated my DiffEqBase master and I now get the same error..  seems to be related to this change:
https://github.com/SciML/DiffEqBase.jl/pull/517/commits/562205d6e5f3522c42b0c11971dc860329aae8c5